packages/web-app-external/src/App.vue

```diff
@@ -0,0 +1,123 @@
+<template>
+  <main
+    class="uk-height-1-1"
+    :class="{
+      'uk-flex uk-flex-center uk-flex-middle': loading || loadingError
+    }"
+  >
+    <h1 class="oc-invisible-sr" v-text="pageTitle" />
+    <loading-screen v-if="loading" />
+    <error-screen v-else-if="loadingError" />
+    <iframe
+      v-if="appUrl && method === 'GET'"
+      :src="appUrl"
+      class="uk-width-1-1 uk-height-1-1"
+      :title="iFrameTitle"
+    />
+    <div v-if="appUrl && method === 'POST' && formParameters">
+      <form :action="appUrl" target="app-iframe" method="post">
+        <input ref="subm" type="submit" :value="formParameters" class="oc-hidden" />
+        <div v-for="(item, key, index) in formParameters" :key="index">
+          <input :name="key" :value="item" type="hidden" />
+        </div>
+      </form>
+      <iframe name="app-iframe" class="uk-width-1-1 uk-height-1-1" :title="iFrameTitle" />
```

Comment:
The `uk-height-1-1` should not be needed, since you have set a min-height of `100vh` in the styles.

Reply:
Good call, mixed that one up with `uk-height-viewport` and try to stick with classes here..